src/js/components/List/List.js

```diff
@@ -337,6 +337,7 @@ const List = React.forwardRef(
               setLastActive(active);
               updateActive(undefined);
             }}
+            onMouseDown={(event) => event.preventDefault()}
```

Comment:
This change breaks the drag and drop re-ordering functionality in list. To prevent the focus indicator from being displayed if we are interacting with the component using our mouse I think we just need to add the following to the StyledList:
```
&:focus:not(:focus-visible) {
    ${unfocusStyle()}
  }
```
See StyledButton for an example

Reply:
ok